lib/typescript/render/providers/google/components/RichCard/Media/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import {MediaHeight} from '../../../googleModel';
+import {ImageWithFallback} from 'render/components/ImageWithFallback';
+
+import styles from './index.module.scss';
+
+export type MediaRenderProps = {
+  height: MediaHeight;
+  contentInfo: {
+    altText?: string;
+    fileUrl: string;
+    forceRefresh: boolean;
+  };
+};
+
+export const Media = ({height, contentInfo: {altText, fileUrl}}: MediaRenderProps) => (
+  <ImageWithFallback
+    src={fileUrl}
+    alt={altText}
+    className={`${styles.mediaImage} ${
+      height === MediaHeight.tall ? styles.tall : height === MediaHeight.medium ? styles.medium : styles.short
```

Comment:
I'd extract this into a `const heightStyle = new Map([MediaHeight.tall, styles.tall], //etc...])` map (or maybe there's even a simpler construct?) and then do `heightStyle.get(height)`

The double ternary reads bad :)